<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemFindDialog</class>
 <widget class="QDialog" name="SystemFindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find System</string>
  </property>
  <widget class="QComboBox" name="comboBoxFind">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>19</y>
     <width>411</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLocation">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>57</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>102</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPattern">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>98</y>
     <width>411</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>*ext1, *ext2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="advancedOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>141</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show advanced options</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowse">
   <property name="geometry">
    <rect>
     <x>456</x>
     <y>58</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxAdvanced">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>178</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Advanced</string>
   </property>
   <widget class="QRadioButton" name="findPlainText">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>32</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find &amp;plain text</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="findRegularExpression">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>60</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find with &amp;regular expressions</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="findSpecialCharachters">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>89</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find &amp;with special characters (\n, \r, \t, \0, \u...,\x...)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="matchWholeWord">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>120</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Match whole word only</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="matchCase">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>150</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Match case</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="includeSubdirectories">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>180</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Include Subdirectories</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>60</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findNext">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>399</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="icon">
    <iconset theme="go-down"/>
   </property>
  </widget>
  <widget class="QPushButton" name="selectAll">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>401</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findPrevious">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="icon">
    <iconset theme="go-up"/>
   </property>
  </widget>
  <widget class="QLabel" name="m_statusLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>440</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="m_progressBar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>437</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
